app/models/user.rb

```diff
@@ -6,6 +6,10 @@ class User < ApplicationRecord
   has_many :services, foreign_key: 'seller_id'
   has_many :offers, foreign_key: 'buyer_id'
   has_one_attached :photo
-  
+
   validates :location, presence: true
+
+  def online?
+    updated_at > 5.minutes.ago
+  end
```

Comment:
Does this mean that user will be seen as offline by other users if he logged in more than 5 minutes ago, but is still browsing the app? 🤔